src/pages/Community.tsx

```diff
@@ -1,13 +1,190 @@
 import WidgetBase from '@dojo/framework/widget-core/WidgetBase';
 import { tsx } from '@dojo/framework/widget-core/tsx';
+import ThemedMixin, { theme } from '@dojo/framework/widget-core/mixins/Themed';
+
+import Grid from '../widgets/grid/Grid';
+import LinkedCard, { LinkedCardProperties } from '../widgets/card/LinkedCard';
 
 import * as css from './Community.m.css';
+import CardHeader, { CardHeaderProperties } from '../widgets/card/CardHeader';
+
+const conduct = require('../assets/icon-conduct.svg');
+const github = require('../assets/logo-github.svg');
+const discord = require('../assets/logo-discord.svg');
+const twitter = require('../assets/logo-twitter.svg');
+const discourse = require('../assets/logo-discourse.svg');
+const dojo = require('../assets/dojo-logo-black.svg');
+
+export interface SimpleLinkedCardProperties extends CardHeaderProperties, LinkedCardProperties {
```

Comment:
I followed the conversation but I'd like to note that this code makes me sad.

Reply:
To expand on the above, I think of the markup portion and the code portion of components as being separate and tend to assume the markup portion is passive or side effect free. I think having a big chunk of data that’s effectively markup is ugly.

I do not have a good idea on an alternative approach and this is the result of some deliberation over slack, which is why I approved.